### PR TITLE
Add test case .+ for zb3

### PR DESCRIPTION
**Pull request by andy1li**
https://secure.helpscout.net/conversation/2747071801/5920?viewId=8414074

https://codecrafters-io.slack.com/archives/C07H9EY1Z6X/p1730129711610299?thread_ts=1730129385.174089&cid=C07H9EY1Z6X

### CODE REVIEW

File: internal/stage_wildcard.go [hunk 1]
```diff
@@ -21,6 +21,11 @@ func testWildcard(stageHarness *test_case_harness.TestCaseHarness) error {
 			Input:            "car",
 			ExpectedExitCode: 1,
 		},
+		{
+			Pattern:          "c.+t",
+			Input:            "coat",
```
Comment (ryan-gang) [inline internal/stage_wildcard.go:26]: I think we should include another case, testing the success scenario. 
```
Pattern: "g.+gol"
Input: "gooooooogol"
```

**Comment by ryan-gang** [comment]
Just wanted Paul's opinion too. 
Requested for his review on this.

File: internal/stage_wildcard.go (hunk 1 above)
Comment (rohitpaulk) [reply internal/stage_wildcard.go:26]: Yep - we should definitely test a success + failure combo when testing something new. Don't want the test list to get too long though (might start hitting timeouts) - @andy1li maybe we could add the case @ryan-gang suggested (feel free to simplify it if you'd like) and remove the 2nd test case? The 2nd one (`c.t` matches `cot`) doesn't seem to test much more than the first one (`c.t` matches `cat`).

Comment (andy1li) [reply internal/stage_wildcard.go]: @ryan-gang How does the updated pair look?

**Pull request merged**